src/data/stig_multipliers.ts

```diff
@@ -1984,6 +2010,72 @@ const DPS_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  Zhenyi: {
+    T: [
+      new Multiplier({
+        active: true,
+        name: "Zhenyi T: lightning DMG",
+        type: Type.EleDealt,
+        value: 45,
+      }),
+    ],
+    M: [
+      new Multiplier({
+        active: true,
+        name: "Zhenyi M",
+        type: Type.TdmDealt,
+        value: 15,
+      }),
+      new Multiplier({
+        active: false,
+        name: "Zhenyi M: On exit, ends [Mitternachts] status to grant team lightning DMG for 18s",
+        type: Type.EleDealt,
+        value: 15,
+      }),
+    ],
+    B: [
+      new Multiplier({
+        active: true,
+        name: "Zhenyi B",
+        type: Type.EleDealt,
+        value: 25,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Zhenyi B: target hit by weapon active takes more lightning DMG",
+        type: Type.EleTaken,
+        value: 20,
+      }),
+    ],
+    "2 set": [
+      new Multiplier({
+        active: true,
+        name: "Zhenyi 2 set",
+        type: Type.TdmDealt,
+        value: 25,
+      }),
+      new Multiplier({
+        active: true,
+        name: "Zhenyi 2 set: for 15s upon switch in, gain lightning DMG",
```

Comment:
The collection in-game shows 12 seconds I think?

Reply:
Yup; originally wrote the changes based off discord translation; only checked the multipliers in collection before making the pull request kek